tested/languages/javascript/templates/value_basic.mako

```diff
@@ -22,8 +22,9 @@
     ])\
 % elif value.type == BasicObjectTypes.MAP:
     {\
-    % for key, item in value.data.items():
-        "${key}": <%include file="statement.mako" args="statement=item" />\
+    % for pair in value.data:
```

Comment:
The numbers and singleton arrays will be casted to strings. Otherwise this will generate a runtime fault.
Do you suggest to raise an error while generating the code?